File: app/src/main/java/kr/co/jayhy/weather/listener/ForecastWeatherCallback.java
package kr.co.jayhy.weather.listener;

import kr.co.jayhy.weather.repo.WeatherForecastRepo;

/**
 * Created by jayhy on 2017. 8. 12..
 */

public interface ForecastWeatherCallback {

    void onForecastWeatherCallback(boolean isSuccess, WeatherForecastRepo result);
}

File: kotlin/src/main/java/kr/co/jayhy/weather/listener/ForecastWeatherCallback.kt
package kr.co.jayhy.weather.listener

import kr.co.jayhy.weather.repo.WeatherForecastRepo

/**
 * Created by jayhy on 2017. 8. 12..
 */

interface ForecastWeatherCallback {

    fun onForecastWeatherCallback(isSuccess: Boolean, result: WeatherForecastRepo)
}
